### ISSUE TITLE
Error: You must provide a Stripe Element or a valid token type to create a Token.

### ISSUE THREAD

**Issue by timsayshey** [bennypowers/stripe-elements#97]
My Vue app is a web component with a shadow dom and I'm using the `stripe-elements` component but after entering the credit card into the form, I grab the `stripe-elements` element and I try to run `stripe.createToken()` and I get the following error:

Here's my app: https://app.ecardwidget.com/widget/5150

![image](https://user-images.githubusercontent.com/1148940/187501989-f6ae81fa-2efc-42a6-a9bb-5edf1211bb29.png)

**Comment by bennypowers** [comment]
Kinda hard to debug this based on just a screenshot, I'm going to need more information. Did you provide a publishable key? How? Can you verify that the element has that key before calling create token?

**Comment by timsayshey** [comment]
Yes, I provided it, just like in the Vue example you provided. Yes, I verified it.

Does this help? 

![image](https://user-images.githubusercontent.com/1148940/187504385-05e724a6-4fcf-456c-8949-7aac67bf72eb.png)
